### PR TITLE
Datica

### PR DESCRIPTION
**Pull request by jault3**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ file or class name and description of purpose be included on the
 same "printed page" as the copyright notice for easier
 identification within third-party archives.
 
-Copyright 2016 Catalyze, Inc.
+Copyright 2016 Datica, Inc.
```
Comment (nathanjsweet) [inline LICENSE.md]: Copyright year okay?

File: commands/ssl/contract.go [hunk 2]
```diff
@@ -93,7 +93,7 @@ var VerifySubCmd = models.Command{
 	Name:      "verify",
 	ShortHelp: "Verify whether a certificate chain is complete and if it matches the given private key",
 	LongHelp: "`ssl verify` will tell you if your SSL certificate and private key are properly formatted for use with Stratum. " +
```
Comment (nathanjsweet) [inline commands/ssl/contract.go]: "Stratum" should be "Compliant Cloud"

File: config/settings.go [hunk 3]
```diff
@@ -36,6 +39,15 @@ func (s FileSettingsRetriever) GetSettings(envName, svcName, accountsHost, authH
 		os.Exit(1)
 	}
 
+	if _, err = os.Stat(filepath.Join(HomeDir, OldSettingsFile)); err == nil {
+		logrus.Debugln("Migrating settings file from .catalyze to .datica")
+		err = os.Rename(filepath.Join(HomeDir, OldSettingsFile), filepath.Join(HomeDir, SettingsFile))
+		if err != nil {
+			logrus.Printf("Error encountered migrating the settings file from .catalyze to .datica: %s", err)
```
Comment (nathanjsweet) [inline config/settings.go]: Do you maybe want to add a little more direction here? You could say something like:
"To fix the issue simply rename the file from $OLD_FILE_PATH to $NEW_FILE_PATH"

**Comment by nathanjsweet** [comment]
LGTM

File: config/settings.go (hunk 3 above)
Comment (jault3) [inline config/settings.go]: Good idea, ill throw that in.

**Pull request merged**